homeassistant/components/light/lifx/__init__.py

```diff
@@ -41,7 +45,10 @@
 
 CONF_SERVER = 'server'
 
+SERVICE_LIFX_SET_COLOR = 'lifx_set_color'
```

Comment:
I guess we can think of other names, set_attributes, set_properties… guess all of them are slightly confusing.